projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-headline/order-detail-headline.component.html

```diff
@@ -24,4 +24,43 @@
       </div>
     </div>
   </div>
+  <div *ngIf="order?.orgCustomer?.active" class="cx-header row">
```

Comment:
We need to figure out the best way fo figure out is the order was placed by an org customer.

Reply:
as discussed, we will be using orgUnit